yarn-project/circuit-types/src/logs/l1_payload/l1_note_payload.ts

```diff
@@ -91,15 +96,18 @@ export class L1NotePayload extends L1Payload {
    */
   public static decryptAsIncoming(ciphertext: Buffer | bigint[], ivsk: GrumpkinScalar) {
     const input = Buffer.isBuffer(ciphertext) ? ciphertext : Buffer.from(ciphertext.map((x: bigint) => Number(x)));
-    const reader = BufferReader.asReader(input);
+    const [publicValues, remainingCiphertext] = this.#getPublicValuesAndRemainingCipherText(input);
```

Comment:
The first argument to this function should no longer be called `ciphertext` because that's not what it is: it'll be a single byte followed by ciphertext, followed by optional non encrypted data. Maybe we call this `content`?

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/8538/commits/0ede13b110186992d93caa6217706b4e83330c40